### PR TITLE
MOTECH-2088: moved MotechListenerType and EventCustomerStarter to new…

### PR DESCRIPTION
**Pull request by sebastianceronik**
… packages for clearance.

### CODE REVIEW

File: platform/event/pom.xml [hunk 1]
```diff
@@ -128,7 +128,9 @@
                             org.motechproject.event;version=${project.version},
                             org.motechproject.event.listener;version=${project.version},
                             org.motechproject.event.listener.annotations;version=${project.version},
-                            org.motechproject.event.messaging;version=${project.version}
+                            org.motechproject.event.listener.annotations.types;version=${project.version},
+                            org.motechproject.event.messaging;version=${project.version},
+                            org.motechproject.event.starter;version=${project.version}
```
Comment (pgesek) [inline platform/event/pom.xml]: Why are exporting this starter package?

File: platform/event/src/main/java/org/motechproject/event/listener/annotations/types/MotechListenerType.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.motechproject.event.listener.annotations;
+package org.motechproject.event.listener.annotations.types;
```
Comment (pgesek) [inline platform/event/src/main/java/org/motechproject/event/listener/annotations/types/MotechListenerType.java]: what's the point of this new package? It just contains 1 class, and both the old one and the new one are still exported.

**Comment by pgesek** [comment]
Do we need org.motechproject.event.messaging exported?

**Comment by sebastianceronik** [comment]
We need to export org.motechproject.event.messaging since classes there are used as helper classes for IT tests, not all though . Removed MotechEventTransformer and MotechEventHeaderMapper

**Comment by pgesek** [comment]
We can go with this for now

**Pull request merged**